### DISCUSSION TITLE
[WinError 10061] No connection could be made because the target machine actively refused it'))

### DISCUSSION THREAD

**Discussion by kartp536** [zowe/zowe-client-python-sdk#161] [Show and tell]
Hi all, 

I tried below Python code to submit the job using ZOWE SDK,

from zowe.zos_console_for_zowe_sdk import Console
from zowe.zos_jobs_for_zowe_sdk import Jobs
connection = {
        "host_url": "XXXX",
        "user": "XXXXX",
        "password": "XXXX",
    }
my_console = Console(connection)
my_jobs = Jobs(connection)

job01 =my_jobs.submit_from_mainframe("XXXXX.JCL.LIB(KC)")
print (job01['jobid'])

and i got the below response, 

raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='XXXX', port=443): Max retries exceeded with url: /zosmf/restjobs/jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EBEADB5D90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))

I checked  my host XXXX , its up and running  but still get the error ,Could you please let me know am i miss anything here ?

**Comment by adam-wolfe** [comment]
Hi @kartp536, you may need to specify the port (if you want to connect to z/OSMF on a port other than 443) in the host_url string, e.g.: `"host_url":"lpar.mainframe.net:1234"`

This is not clear from the existing documentation.

**Reply by kartp536** [reply]
@adam-wolfe  Thanks, Looks like the previous error is fixed but facing the new message " 
requests.exceptions.SSLError: HTTPSConnectionPool(host='XXXXXXXX.com', port=11443): Max retries exceeded with url: /zosmf/restjobs/jobs/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1131)')))"   

I tried "https://community.ibm.com/community/user/ibmz-and-linuxone/blogs/sheng-jie-han/2021/06/03/how-to-fix-certificate-verify-failed-self-signed-c" but no luck. am I missing anything here?

**Reply by adam-wolfe** [reply]
I'm not sure if this will work, but you could try adding a line to the connection definition to disable SSL verification:

```
connection = {
    "host_url": "XXXX:1234",
    "user": "XXXXX",
    "password": "XXXX",
    "ssl_verification": False
}
```

**Reply by kartp536** [reply]
Thank you @adam-wolfe , finally it works.
